src/torchmetrics/functional/text/sacre_bleu.py

```diff
@@ -127,9 +153,9 @@ def __call__(self, line: str) -> Sequence[str]:
         return self._lower(tokenized_line, self.lowercase).split()
 
     @classmethod
-    def tokenize(
-        cls, line: str, tokenize: Literal["none", "13a", "zh", "intl", "char"], lowercase: bool = False
-    ) -> Sequence[str]:
+    def tokenize(cls, line: str, tokenize: Tokenizers, lowercase: bool = False) -> Sequence[str]:
```

Comment:
```suggestion
    def tokenize(cls, line: str, tokenize: _Tokenizer_list, lowercase: bool = False) -> Sequence[str]:
```
let's use different name so it does not collide with HF names and keep it private